src/App.js

```diff
@@ -1,10 +1,79 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-const App = () => (
-  <div>
-    <h2>Todo App</h2>
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
 
-  </div>
-);
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      inputTextValue: '',
+      todoListTasks: [],
+    };
+  }
+
+  // submit new task
+  handleAddTask = () => {
+    if (!this.state.inputTextValue) {
+      alert('No Task!');
+    }
+    else {
+      this.setState({
+        inputTextValue: '',
+        todoListTasks: [
+          {
+            task: this.state.inputTextValue,
+            id: Date.now(),
+            completed: false,
+            ID: Date.now(),
+          },
+          ...this.state.todoListTasks
+        ]
+      });
+    }
+  };
+
+  // handle text input value
+  handleInputChange = e => {
+    this.setState({
+      inputTextValue: e.target.value,
+    });
+  }
+
+  // handle task completion (remove task)
+  handleTaskCompletion = e => {
+    const taskID = e.target.id;
+    const stateArr = [...this.state.todoListTasks];
+    
+    for (let i = 0; i < stateArr.length; i++) {
+      if ('' + stateArr[i].ID === taskID) {
+        stateArr.pop();
```

Comment:
This seems wrong... yes, this will only remove the last most item on the list regardless of the target.
Perhaps simply setting display: none; on stateArr[i] would suffice.

Reply:
This would require some kind of handler, perhaps another field in the state array entry properties.